### PR TITLE
Fixes #2643

### LINKED ISSUE #2643: Chaplain Hood can't Hold Bible in suit storage.
#### Additional info: I was told that it is intended for bibles to fit in Chaplain Hoodies
 - **Server Revision**: Found using the "Show Server Revision" verb under the OOC tab.
 - **Anything else you may wish to add** (Location if it's a mapping issue, etc)

### PR DESCRIPTION
**Pull request by Atermonera**
Also unifies list() grammar within \code\modules\clothing\suits\jobs.dmi for better readability.

### CODE REVIEW

**Comment by Yoshax** [comment]
For future reference, as far as I am aware you don't need the \ to indicate a new line or such, or to split your list over multiple lines.

**Comment by Atermonera** [comment]
Ok. Saw the format in a couple instances within the file, so I just copied that.

File: code/modules/clothing/suits/jobs.dm [hunk 1]
```diff
@@ -107,7 +107,7 @@
 	desc = "A forensics technician jacket."
 	body_parts_covered = UPPER_TORSO|ARMS
 	allowed = list(/obj/item/weapon/tank/emergency_oxygen, /obj/item/device/flashlight, /obj/item/weapon/gun/energy, /obj/item/weapon/gun/projectile, /obj/item/ammo_magazine, \
-	/obj/item/ammo_casing, /obj/item/weapon/melee/baton, /obj/item/weapon/handcuffs, /obj/item/device/taperecorder, /obj/item/device/uv_light)
+	/obj/item/ammo_casing, /obj/item/weapon/melee/baton, /obj/item/weapon/handcuffs, /obj/item/weapon/storage/fancy/cigarettes, /obj/item/weapon/flame/lighter/obj/item/device/taperecorder, /obj/item/device/uv_light)
```
Comment (Yoshax) [inline code/modules/clothing/suits/jobs.dm]: Missing a comma after lighter.

**Comment by Anewbe** [comment]
Conflicts

**Comment by Atermonera** [comment]
Ok, because I'm still learning Git and Anewbe made a good suggestion, closing this to sort things out on local branches, and I'll make a new PR. The conflict is here because I made my changes, and then made the commits a week later, and in that time, #2672 got merged.

**Pull request closed without merging**